CommonUtils/src/main/java/com/bnnthang/fltestbed/commonutils/models/TrainingConfiguration.java

```diff
@@ -68,14 +68,12 @@ public class TrainingConfiguration {
     private ArrayList<ArrayList<Double>> distributionRatiosByLabels;
 
     /**
-     * If clients should be dropped
+     * Ture if clients should be dropped
```

Comment:
This is not what we intended. We want the weights to be the same as the data partition ratios.